Makefile

```diff
@@ -7,6 +7,7 @@ IMG ?= ghcr.io/bank-vaults/vault-operator:dev
 TEST_K8S_VERSION ?= 1.27.1
 TEST_VAULT_VERSION ?= 1.13.3
 TEST_BANK_VAULTS_VERSION ?= 1.19.0
+TEST_BANK_VAULTS_IMAGE ?= ghcr.io/banzaicloud/bank-vaults:$(TEST_BANK_VAULTS_VERSION)
```

Comment:
The latest version of bank vaults does not work, only a fixed 1.19.0 version

Reply:
note: it requires additional security configuration which is not relevant for local operator development

Reply:
bank-vaults should be fixed by [this](https://github.com/banzaicloud/bank-vaults/pull/1903) once its merged btw

Reply:
awesome @akijakya, can we have a ticket for this?

Reply:
thanks @akijakya, i have made a separate issue for this